roles/girder-service-install/meta/main.yml

```diff
@@ -8,6 +8,5 @@ dependencies:
     group: "{{ girder_group }}"
     description: Girder Data Management Platform -- Web Service
     command: >
-        bash -c "cd {{ girder_install_root }} && python -m girder"
+        bash -c ". {{ girder_venv_root }}/bin/activate && cd {{ girder_install_root }} && python -m girder"
```

Comment:
I didn't realize I used `&&` for compound commands, but now that I think about it, I'd much rather we used `;` to ensure we don't run into anything funny with subshells.

```
bash -c ". {{ girder_venv_root }}/bin/activate ; cd {{ girder_install_root }} ; python -m girder"
```